plugins/initialization-control/src/main/java/org/qbicc/plugin/initializationcontrol/QbiccFeatureProcessor.java

```diff
@@ -27,17 +28,13 @@ public static void process(CompilationContext ctxt, List<Path> features) {
         mapper.setVisibility(PropertyAccessor.FIELD, JsonAutoDetect.Visibility.ANY);
         mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
 
-
-        for (Path feature: features) {
+        for (URL feature: features) {
             QbiccFeature qf;
             try {
-                InputStream fs = new FileInputStream(feature.toFile());
+                InputStream fs = feature.openStream();
```

Comment:
I guess I missed this last time but the stream needs to be closed; it should go in a try-with-resources block.

Reply:
Thanks.  Fixed.